shoulder/__init__.py

```diff
@@ -26,6 +26,7 @@
 from shoulder.logger import logger
 from shoulder.config import config
 
+import shoulder.access_attributes
```

Comment:
You are right, this is awkward, and it is actually redundant since it is imported in `armv8_xml_parser.py` which I think is a more appropriate place to import anyways.